shell/browser/electron_browser_client.cc

```diff
@@ -47,6 +47,8 @@
 #include "content/public/common/web_preferences.h"
 #include "electron/buildflags/buildflags.h"
 #include "electron/grit/electron_resources.h"
+#include "extensions/browser/api/web_request/web_request_api.h"
```

Comment:
Headers must be behind `#if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)`

Reply:
Is it good now?

Reply:
yup thanks!